src/Hazelcast.Net/DistributedObjects/ReliableTopicEventHandler.cs

```diff
@@ -48,25 +48,47 @@ public ReliableTopicEventHandler<T> Message(Func<IHReliableTopic<T>, ReliableTop
         return this;
     }
     
-    // Disposed event is single, no chaining.
+    /// <summary>
+    /// Sets the handler which runs after the subscription disposed or canceled.
+    /// </summary>
+    /// <param name="handler">The handler.</param>
+    public ReliableTopicEventHandler<T> Terminated(Action<IHReliableTopic<T>, ReliableTopicMessageEventArgs<T>> handler)
```

Comment:
and, should this be a `ReliableTopicMessageEventArgs` or ... ? there's no message here ?

Reply:
Having a separated argument class is better, you are right. Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/825/commits/29bfc61d64d9fcf6984c77fe142aac9fa8d43173